frontend/src/views/toolbox/clean/index.vue

```diff
@@ -13,7 +13,7 @@
             <template #main>
                 <el-row class="mt-5 mb-5">
                     <el-col :span="1"><br /></el-col>
-                    <el-col :xs="24" :sm="20" :md="20" :lg="10" :xl="10">
+                    <el-col :xs="24" :sm="20" :md="20" :lg="14" :xl="14">
                         <div v-if="scanStatus !== 'scanned'">
                             <div v-if="scanStatus === 'beforeScan'">
                                 <div v-if="form.lastCleanTime">
```

Comment:
The main difference that might be of note is the change from `:lg="10"` to `:lg="14"`. This could potentially affect the layout on larger screens (such as lg tablets or desktops). It's always good to review spacing adjustments across different screen sizes. If this is intentional, make sure it aligns with your design goals.